src/helpers/providerIDs.ts

```diff
@@ -2,23 +2,23 @@ import MyDataHelps from "@careevolution/mydatahelps-js";
 
 export function getFitbitProviderID() {
 	var fitbitProviderID = 564;
-	if (MyDataHelps.baseUrl &&  MyDataHelps.baseUrl.startsWith("https://mdhorg.ce.dev")) {
+	if (MyDataHelps.baseUrl && (MyDataHelps.baseUrl.startsWith("https://mdhorg.ce.dev") || MyDataHelps.baseUrl.startsWith("https://designer.mydatahelps.dev"))) {
```

Comment:
I would think this mostly matters for mydatahelps.dev, not designer.mydatahelps.dev?

Reply:
🤦